servo.ts

```diff
@@ -31,6 +31,26 @@ namespace crickit {
             dev.setPwmFreq(this._pin, 50);
             dev.analogWrite(this._pin, value);
         }
+
+        /**
+         * set the pulse width to the servo in microseconds
+         */
+        //% group="Servos"
+        //% weight=10
+        //% blockId=sawservosetpulse block="crickit set %servo pulse to (μs) %value"
```

Comment:
you can move (us) after %value

Reply:
ok will do (was matching the format to the corresponding block under the Pins -> Servo heading)

Reply:
Right... we've improved our block format parser now and it can handle a proper ordering of words.